src/components/LogRow/BaseRow/index.tsx

```diff
@@ -104,7 +104,7 @@ const BaseRow: React.FC<BaseRowProps> = ({
       onDoubleClick={handleDoubleClick}
       shared={shared}
     >
-      <StyledIcon
```

Comment:
Tbh I never knew what the function of this was button until now 😂 what do you think of adding a `title` attribute explaining it?

Reply:
I gave it a try but I don't think LG supports the `title` attribute properly, because when I hover no tooltip text ever appears =( I think the title text needs to be embedded in the original SVG, so it's hard to work around

Reply:
Tracked by [LG-3416](https://jira.mongodb.org/browse/LG-3416)